release-content/0.15/release-notes/15276_Reduce_runtime_panics_through_SystemParam_validation.md

```diff
@@ -1,4 +1,60 @@
 <!-- Reduce runtime panics through `SystemParam` validation -->
 <!-- https://github.com/bevyengine/bevy/pull/15276 -->
 
-<!-- TODO -->
+<!-- `QuerySingle` family of system params -->
+<!-- https://github.com/bevyengine/bevy/pull/15476 -->
+
+<!-- `Populated` (query) system param -->
+<!-- https://github.com/bevyengine/bevy/pull/15488 -->
+
+
+**Bevy 0.15** comes with a much needed ergonomic improvement in inaccessible system parameters named *Fallible System Params*.
+Originally, the following piece of code would panic:
+```rust
+#[derive(Resource)]
+struct MyResource;
+
+fn my_system(my_resource: Res<MyResource>) {}
+
+fn main() {
+    let mut app = App::new();
+    app.add_plugins(DefaultPlugins)
+    app.add_systems(my_system);
+    // Panic here: `my_system` cannot fetch `MyResource`, because it was never added.
+    app.run();
+}
+```
+but now, `my_system` simply won't be executed.
+
+This works on all system-based features:
+- Systems & Observers - will be skipped,
+- Run conditions - will be skipped and return `false`.
+
+Behavior in compound systems, like `system_a.pipe(system_b)`, is still in development.
```

Comment:
```suggestion
Compound systems, like `system_a.pipe(system_b)`, are currently skipped if any required data is missing.
```

Reply:
There is a lot of complexity here, but let's say this is how it works >.>